apps/web/src/stores/editor-store.js

```diff
@@ -145,6 +147,10 @@ class EditorStore extends BaseStore {
         attachmentsLength: db.attachments.ofNote(note.id, "all")?.length || 0
       };
     });
+    if (!db.notes.note(noteId))
```

Comment:
This will be extremely fragile. It's better to create a constant `isDeleted` that holds the value. Trashed notes have a `dateDeleted` property & their type is also `trash` so that will be more reliable.